### ISSUE TITLE
Implement ability for missing mods prompt to factor dependencies of missing mods

### ISSUE THREAD

**Issue by twstagg** [RimSort/RimSort#144]
**Is your feature request related to a problem? Please describe.**
Missing mods prompt currently does not try to calculate dependencies for missing mods

**Describe the solution you'd like**
Missing mods prompt should calculate dependencies and add them to the download list

**Describe alternatives you've considered**
Current behavior does not do this. This does need done.

**Additional context**
This information can be ascertained with external metadata

**Comment by LionelColaso** [comment]
incase you want inspiration there's a mod on workshop that does the same thing
https://steamcommunity.com/sharedfiles/filedetails/?id=2805177535

**Comment by LionelColaso** [comment]
fixed in #730
